checks/binary_artifact.go

```diff
@@ -98,6 +98,13 @@ func checkBinaryFileContent(path string, content []byte,
 		return false, sce.WithMessage(sce.ErrScorecardInternal, fmt.Sprintf("filetype.Get:%v", err))
 	}
 
+	if strings.Contains(strings.ToLower(path), "test") {
```

Comment:
We have `isTestData` fn already which looks for test files - https://github.com/ossf/scorecard/blob/main/checks/file_utils.go#L51. Maybe use that here instead?

Reply:
It can't be used directly because it looks for "testdata" files only. It's possible to relax it by letting it skip "test" files but I'm not sure it's a good idea in the sense that I wouldn't want to start filtering out more files than I should in other places just because of the Binary-Artifacts check. I'd characterize this PR as a stop gap that should probably even look like a stop gap :-)

Reply:
Makes sense. We can submit this as is for now.